src-vue/src/views/DeveloperView.vue

```diff
@@ -29,6 +34,28 @@ export default defineComponent({
                 type: 'error',
                 position: 'bottom-right'
             });
+        },
+        async toggleReleaseCandidate() {
+            // Flip between RELEASE and RELEASE_CANDIDATE
+            if (this.$store.state.release_canal === ReleaseCanal.RELEASE) {
+                this.$store.state.release_canal = ReleaseCanal.RELEASE_CANDIDATE;
+            }
+            else {
+                this.$store.state.release_canal = ReleaseCanal.RELEASE;
+            }
+
+            // Update current state so that update check etc can be performed
+            this.$store.commit("initialize");
+
+            console.log(this.$store.state)
+
+            // Display notification to highlight change
+            ElNotification({
+                title: `${this.$store.state.release_canal}`,
+                message: `Switched release channel to: "${this.$store.state.release_canal}"`,
+                type: 'error',
```

Comment:
I don't think this is an error message; you should use [success or info type](https://element-plus.org/en-US/component/notification.html#with-types) instead.

Reply:
Whoops forgot to update from testing, fixed in 0085a57a3d35c340552859a8d8ae95df6ba5e8b8